### PR TITLE
Initial jooq flyway commit to connect to job activity

### PR DESCRIPTION
**Pull request by amitaekbote**
still a work in progress

### CODE REVIEW

File: titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RDSSSLSocketFactory.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import java.security.KeyStore;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.postgresql.ssl.WrappedFactory;
+
+public class RDSSSLSocketFactory extends WrappedFactory {
+    public RDSSSLSocketFactory() {
+        try {
+            KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
```
Comment (tbak) [inline titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RDSSSLSocketFactory.java:29]: This code is duplicated (we should call into RdsUtil added below).

File: titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+public final class RdsUtils {
+    /**
+     * Setting as a constant as it is not critical.
+     */
+    private static final String TRUST_STORE_PASSWORD = "titus123";
+
+    public static SSLSocketFactory createRdsSSLSocketFactory() {
+        try {
+            KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
+            try (InputStream inputStream = RdsUtils.class.getClassLoader().getResourceAsStream("RDS-2019.truststore")) {
```
Comment (tbak) [inline titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java:34]: For security reason, we should allow injecting the truststore file name and the password here. Thus internally we can use a different file, and a different (ideally encrypted) password.

Comment (tbak) [reply titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java:34]: We should reference the attached truststure in this repo for running the integration tests only.

File: titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RDSSSLSocketFactory.java (hunk 1 above)
Comment (amitaekbote) [reply titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RDSSSLSocketFactory.java:29]: fixed it.

File: titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java (hunk 2 above)
Comment (amitaekbote) [reply titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java:34]: will fix this in the following pr, finally able to test this. was fighting gradle builds. definitely needs more work here

**Pull request merged**